ekncontent/Makefile.am

```diff
@@ -163,8 +163,16 @@ javascript_tests = \
 	tests/ekncontent/testXapianBridge.js \
 	$(NULL)
 
+test_content = \
+	tests/testcontent/ekn/share/com.endlessm.fake_test_app.en/EKN_VERSION \
+	tests/testcontent/ekn/share/com.endlessm.fake_test_app.en/subscritions.json \
```

Comment:
typo: subscritions -> subscriptions (unlike the other typo, this one has consequences :smile:)